website/source/docs/providers/aws/r/opsworks_stack.html.markdown

```diff
@@ -37,6 +37,7 @@ The following arguments are supported:
 * `service_role_arn` - (Required) The ARN of an IAM role that the OpsWorks service will act as.
 * `default_instance_profile_arn` - (Required) The ARN of an IAM Instance Profile that created instances
   will have by default.
+* `agent_version` - (Optional) If set to `"LATEST"`, OpsWorks will automatically install the latest version.
```

Comment:
Hi Paul!

The SDK also allows you to pass specific version numbers there too, but that feature is already exposed at the instance level. The stack being the parent of the instance, will set the instance version by default. The reason for the PR is that _automatic updates_ are not allowed at the instance level (and it's not a default).

You can find supported versions for your stack from the cli:

```json
$ aws opsworks describe-agent-versions --stack-id ***************************************

{
    "AgentVersions": [
        {
            "Version": "3421-20150611173115",
            "ConfigurationManager": {
                "Version": "11.10",
                "Name": "Chef"
            }
        },
        {
            "Version": "3422-20150629124612",
            "ConfigurationManager": {
                "Version": "11.10",
                "Name": "Chef"
            }
        },
        {
            "Version": "3424-20150709092700",
            "ConfigurationManager": {
                "Version": "11.10",
                "Name": "Chef"
            }
        },

```

~ Justin 🍻